configure.ac

```diff
@@ -91,9 +78,8 @@ AX_VALGRIND_CHECK
 
 PKG_PROG_PKG_CONFIG([0.24])
 
-AC_CHECK_FUNCS(fdwalk)
+# For libglnx
 AC_CHECK_HEADER([sys/xattr.h], [], [AC_MSG_ERROR([You must have sys/xattr.h from glibc])])
```

Comment:
Since it's for libglnx, I wonder why is this check not in libglnx itself?

Reply:
Mainly because libglnx doesn't have a complete build system, only a fragment of one.

I actually did start to move this into `libglnx.m4`, but then I looked at libglnx's `meson.build` and noticed that it had never actually checked for this...

In practice, "your glibc doesn't have `sys/xattr.h`" is probably synonymous with "your glibc is from the distant past", so maybe we can just lose this check? The oldest glibc I have convenient access to is eglibc 2.15 from Ubuntu 12.04 (2012), which does have it. Apparently musl also has it.

Reply:
If a system as old as 2012 has it, then it's probably fairly safe to drop the check entirely.